### ISSUE TITLE
Unexpected token export

### ISSUE THREAD

**Issue by tswaters** [formatjs/react-intl#1390]
### Expected behavior

It is possible to require the module using `require`:

### Current behavior

A SyntaxError is thrown,

```
$ node index.js
```

### Step to reproduce for BUG REPORT

```
$ echo "require('react-intl')" > index.js
$ npm i react-intl@next
+ react-intl@3.0.0-beta.23
added 21 packages from 44 contributors in 2.405s
$ node index.js
/home/tyler.waters/code/test/node_modules/intl-messageformat/core.js:1
export * from './lib/formatters';
^^^^^^

SyntaxError: Unexpected token export
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/tyler.waters/code/test/node_modules/react-intl/dist/index.js:9:14)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
```

### Your Environment

|          Executable | Version |
| ------------------: | :------ |
| `npm ls react-intl` | 3.0.0-beta.23 |
|      `npm ls react` | 16.8.6 |
|     `npm --version` | 6.9.0 |
|    `node --version` | 10.16.1 |

| OS   | Version |
| ---- | ------- |
| fedora | 28 |

**Comment by longlho** [comment]
Thanks for reporting. Using ESM import is highly recommended and we'll take a look asap.

**Comment by longlho** [comment]
fixed in 223d2cf429889f05bc01231c83e110d15b603237
